octoprint_rtmpstreamer/__init__.py

```diff
@@ -408,6 +413,9 @@ def _build_overlay(self):
 
         # name, path, size, origin, date
         fileInfo = current_data["job"]["file"]
+        filename = "None Loaded"
+        if fileInfo["name"]:
+            filename = fileInfo["name"].replace(".gcode", "")
```

Comment:
I think it's possible to have both .gcode and .gco file extensions as well as capitalization differences (.GCO, .GCODE).

Reply:
this is a very valid I assume the best here is 4 replaces to make sure, or should I just detect the last . and drop from there?